source/Pacioli/Pacioli.Lib/Models/Review.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Pacioli.Lib.Models
+{
+    public record Review
+    {
+        public Review(string userId, bool approved, string comment) : this(userId, approved)
+        {
+            if (string.IsNullOrEmpty(comment))
```

Comment:
I think we shouldn't require comments, just userId and approval should be enough.